src/components/VmDetails/cards/OverviewCard/style.css

```diff
@@ -46,3 +50,11 @@
   line-height: 20px;
   padding-left: 10px;
 }
+
+.pool-vm-label {
```

Comment:
`OverviewCard` should have __sharedStyles.css__ imported, so there is no need to redefine the style here.